src/gtest/test_noteencryption.cpp

```diff
@@ -100,9 +100,10 @@ TEST(noteencryption, api)
 
 uint256 test_prf(
     unsigned char distinguisher,
-    uint256 x,
+    uint252 seed_x,
     uint256 y
 ) {
+    uint256 x = seed_x.inner();
     *x.begin() &= 0x0f;
```

Comment:
This should no longer be necessary if `uint252` does what I expect. Perhaps change it to:

```c
assert((*x.begin() & 0x0f) == *x.begin());
```